src/toil/batchSystems/mesos/batchSystem.py

```diff
@@ -393,16 +401,23 @@ def resourceOffers(self, driver, offers):
                 runnableTasks.extend(runnableTasksOfType)
             # Launch all runnable tasks together so we only call launchTasks once per offer
             if runnableTasks:
+                unableToRun = False
                 driver.launchTasks(offer.id, runnableTasks)
                 for task in runnableTasks:
                     self._updateStateToRunning(offer, task)
                     log.info('Launched Mesos task %s.', task.task_id.value)
             else:
-                log.info('Although there are queued jobs, none of them could be run with offer %s '
-                         'extended to the framework. Enable debug logging to see details '
-                         'about tasks queued and offers made.', offer.id)
+                log.debug('Although there are queued jobs, none of them could be run with offer %s '
```

Comment:
You are telling the user to enable debug logging in a debug statement. Did you mix up the messages for this and for the `log.info` statement?